### PR TITLE
Added helper class for 2D visual data

### PR DESCRIPTION
**Pull request by rowandempster**

### CODE REVIEW

**Review by k78ma — APPROVED** [review]
LGTM!

File: src/PGP/datasets/visual_data_helper.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        return views
+    
+    def load_visuals(self, data: Dict):
+        if self.transform is None:
+            return data
+        visuals = data['inputs']['target_agent_visuals']
+        imgs = [Image.open(f"{self.nusc.dataroot}/{v['cam_file']}") for v in visuals]
+        imgs = [img.convert("RGB") for img in imgs]
+        data['inputs']['target_agent_images'] = \
+            torch.stack([self.transform(img) for img in imgs])
```
Comment (rowandempster) [inline src/PGP/datasets/visual_data_helper.py]: Since we have a variable number of images (might not have the full 2 second history of the agent), this returns a non-constant sized tensor (in its first dim)...and thus the resulting datapoints cannot be batched (the error that we ran into last night).

How the PGP author got around this (since he ran into the same thing with his non-constant size geometry feature sequences) is to pad them with zeros, and include a `valid` mask. I'm going to look into if we can apply the same technique to get around our issue.
